### DISCUSSION TITLE
Pause module: Relative to the completion of the previous survey

### DISCUSSION THREAD

**Discussion by alpkaanaksu** [rubenarslan/formr.org#547] [Q&A] [answered]
We are creating a run for a diary study, and we'd normally send reminders at a fixed time every day. But for testing purposes, we'd like to move to the next module after a short delay. (e.g. 2 minutes)

What should I type into the 'relative to' field of the pause module?

**Comment by rubenarslan** [comment]
Nothing, just leave it empty.

**Reply by alpkaanaksu** [reply]
This was my first assumption too, but the message is only sent after I refresh the page. We try to send SMS reminders using the `text_message_twilio` function. Are pauses/waits only meant to be used in combination with the email module? I suspect this because of the description on the cron jobs page: 

> The cron job runs every x minutes, to evaluate whether somebody needs to be sent a mail. This usually happens if a pause is over.

**Reply by rubenarslan** [reply]
Which instance of formr are you using? formr.org or the one at LMU OSC? After a pause expires, the message should be automatically sent unless you've disabled "cron" in the study settings. There shouldn't be a difference to how emails are sent.

**Reply by alpkaanaksu** [reply]
I am using the OSC instance at https://tellmi.psy.lmu.de/formr

The option 'Enable cron.' in the settings is checked.

**Reply by rubenarslan** [reply, marked as answer]
I didn't set this instance up. Presumably @nicebread controls it. I suspect the cron job/supervisor daemon is not running. So, your study is set up correctly, but this server is not.

**Reply by alpkaanaksu** [reply]
What do we (in this case Felix with his admin rights) have to do to fix it? Is it documented somewhere?

**Comment by nicebread** [comment]
The http://tellmi.psy.lmu.de/formr/ is not working 100%, and will be shut down. @alpkaanaksu, I'll give you an account on the proper https://formr.osc.lmu.de instance.

**Reply by alpkaanaksu** [reply]
@rubenarslan our SMS reminders work just fine with the proper OSC instance. Thanks for your help!
